### PR TITLE
Bug 1221630 - Fix failure in test_settings_airplane_mode.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e5776b1256f0ff538074a1c19d86d3a2709d93dc

File: tests/python/gaia-ui-tests/gaiatest/form_controls/binarycontrol.py [hunk 1]
```diff
@@ -47,8 +51,16 @@ def is_checked(self):
...
+        # Change for a more standard method, once Bug 1113742 lands
+        return self.marionette.execute_script('return arguments[0].wrappedJSObject.disabled == false', [self.root_element])
 
 class HtmlBinaryControl(BinaryControl):
     @property
     def is_checked(self):
         return self.root_element.is_selected()
+
+    @property
+    def is_checked(self):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/form_controls/binarycontrol.py]: Nit: `is_enabled`

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/form_controls/binarycontrol.py]: Oops

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=94274013e2fdb3ca96e50ef8f9e38edd3f75c5b8

**Pull request merged**